packages/api/src/share_link/service.ts

```diff
@@ -69,4 +74,141 @@ const createShareLink = async (
   };
 };
 
-export const shareLinkService = { createShareLink };
+const getShareLinkColumnValuesForUpdate = (
+  requestData: UpdateShareLinkRequest
+): UpdateableShareLinkColumns => {
+  const columnsToUpdate: UpdateableShareLinkColumns = {};
+  if (requestData.permissionsLevel !== undefined) {
+    columnsToUpdate.defaultaccessrole = `access.role.${requestData.permissionsLevel}`;
```

Comment:
I'd expect camelCase here `defaultAccessRole` -- or maybe this is just the existing column name / you don't get to decide the name in which case please carry on!

Reply:
This is about matching the column name, but I think the need for that will dissipate if I handle the update query as you suggest below.